docs/API.md

```diff
@@ -451,77 +504,144 @@ New version:
+specified as:
+
+* X-... : specifies only a minimum value
+* X-Y,Z : values between X and Y are allowed, and additionally Z
+
+If the value is of `string` type, this field will contain its maximum size
+in square brackets.
+
+If the range contains additionally an asterisk, it means that more elaborate
+restrictions on the value apply, as explained in the description.
+
+7. **Dir**
+
+Option direction: W if can be set, R if can be retrieved, RW if both.
+
+6. **Entity**
+
+This describes whether the option can be set on the socket or the group.
+The G and S options may appear together, in which case both possibilities apply.
+The D and I options, mutually exclusive, appear always with G. 
+The + marker can only coexist with GS.
+
+Possible specifications are:
+
+* S: This option can be set on a single socket (exclusively, if not GS)
+
+* G: This option can be set on a group (exclusively, if not GS)
+
+* D: If set on a group, it will be derived by the member socket
+
+* I: If set on a group, it will be taken and managed exclusively by the group
 
-Note that options are usually set either on a group or on a socket. When set
-on a group, the options will persist. They will be derived by every socket
-automatically created and added to the group. This applies to every
-socket option, unless a different rule is declared explicitly for a specific case.
+* +: This option is also allowed to be set individually on a group member
+socket through a configuration object in `SRT_SOCKGROUPCONFIG` prepared by
+`srt_create_config`. Note that this setting may override the setting derived
+from the group.
 
-This option list is sorted alphabetically. Note that some options can be
-either only a retrieved (GET) or specified (SET) value.
+This option list is sorted alphabetically.
 
 
-| OptName               | Since | Binding | Type  | Units  | Default  | Range  |
-| --------------------- | ----- | ------- | ----- | ------ | -------- | ------ |
-| `SRTO_CONNTIMEO`      | 1.1.2 | pre     | `int` | msec   | 3000     | tbd    |
+| OptName            | Since | Binding |   Type    | Units  | Default  | Range  | Dir | Entity |
+| ------------------ | ----- | ------- | --------- | ------ | -------- | ------ | --- | ------ |
+| `SRTO_CONNTIMEO`   | 1.1.2 | pre     | `int32_t` | msec   | 3000     | 0..    | W   | GSD+   |
 
-- Connect timeout. SRT cannot connect for RTT > 1500 msec (2 handshake exchanges) 
-with the default connect timeout of 3 seconds. This option applies to the caller 
-and rendezvous connection modes. The connect timeout is 10 times the value set 
-for the rendezvous mode (which can be used as a workaround for this connection 
-problem with earlier versions)
+- Connect timeout. This option applies to the caller and rendezvous connection
+modes. The connect timeout is 10 times the value set for the rendezvous mode
+(which can be used as a workaround for this connection problem with earlier
+versions).
```

Comment:
> The connect timeout is 10 times the value set for the rendezvous mode ...

Not very clear which rendezvous option is meant here.

Reply:
There's only one `SRTO_RENDEZVOUS` option in SRT.

Reply:
So, if `SRTO_RENDEZVOUS` is set, then the `SRTO_CONNTIMEO` has a different value? What is the meaning of the sentence?

Reply:
If `SRTO_RENDEZVOUS` is set and `SRTO_CONNTIMEO` is set to default 3000ms, the effective connection timeout will be after 30s.

Reply:
Maybe this way then?

Connect timeout. This option applies to the caller and rendezvous connection
modes. For the rendezvous mode (see `SRTO_RENDEZVOUS`) the effective connection timeout
will be 10 times the value set with `SRTO_CONNTIMEO`.

Reply:
Yes, definitely better.